### PR TITLE
Denmark marriage tool update - Content change request

### PR DESCRIPTION
**Pull request by pmanrubia**
Supersedes #2520 

### Trello card
* [125-denmark-marriage-tool-update-content-change-request](https://trello.com/c/9mdERL9m/125-denmark-marriage-tool-update-content-change-request)

## Description
This story is linked to this [Zendesk ticket](https://govuk.zendesk.com/agent/tickets/1317289). 

Updates outcomes copy where a citizen lives in Uk or in Denmark [according to this document](https://docs.google.com/document/d/1QMLTHznjIMN7__cMhlp_4GCrTMa5MNgkUFHaOMLu04U/).

## Factcheck
[denmark-marriage-tool-update-content-change-request](https://smart-answers-pr-2520.herokuapp.com/marriage-abroad)

## Expected changes (1)

* [URL on GOV.UK](https://gov.uk/marriage-abroad/y/denmark/uk/partner_local/same_sex)

### Before

screen shot 2016-06-07 at 17 13 58

### After

screen shot 2016-06-07 at 17 13 48

## Expected changes (2)

* [URL on GOV.UK](https://gov.uk/marriage-abroad/y/denmark/denmark/partner_local/same_sex)

### Before

screen shot 2016-06-07 at 17 17 49

### After

screen shot 2016-06-07 at 17 13 48

### CODE REVIEW

File: lib/smart_answer_flows/marriage-abroad/outcomes/outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark.govspeak.erb [hunk 1]
```diff
@@ -62,6 +70,13 @@
 
   <%= render partial: 'names_on_documents_must_match.govspeak.erb' %>
 
+  <% if calculator.resident_of_ceremony_country? %>
```
Comment (chrisroos) [inline lib/smart_answer_flows/marriage-abroad/outcomes/outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark.govspeak.erb]: This change seems unrelated to this commit. Moving this code only affects outcomes where the user is a resident of Denmark and this commit is specifically about users that are resident in the UK.

Should it be part of efd5abfb7846589e687054b3a93c6a35d3c58478 or 52297aea298457e754b2768847c1809e345f1ccc instead?

File: lib/smart_answer_flows/marriage-abroad.rb [hunk 2]
```diff
@@ -175,6 +175,8 @@ def define
               outcome :outcome_opposite_sex_marriage_in_poland_when_residing_in_uk_or_poland
             elsif calculator.ceremony_country == 'slovenia'
               outcome :outcome_opposite_sex_marriage_in_slovenia_when_residing_in_uk_or_slovenia
+            elsif calculator.ceremony_country == 'denmark' && (calculator.resident_of_uk? || calculator.resident_of_ceremony_country?)
+              outcome :outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark
```
Comment (chrisroos) [inline lib/smart_answer_flows/marriage-abroad.rb]: This is duplicated for opposite and same sex. I think I'd be inclined to move it outside of the checks for opposite and same sex so that it only occurs once.

**Comment by chrisroos** [comment]
It looks as though "outcome_ceremonies_in_denmark_when_residing_in_uk_or_denmark.govspeak.erb" could be simplified quite a bit. It looks as though there are quite a few conditionals that can be combined.

**Comment by chrisroos** [comment]
I've added a couple of comments but I don't think any of them should prevent this change from being merged. Good work, @pmanrubia!

**Comment by chrisroos** [comment]
I've just spotted a couple of changes to the artefacts that I'm not sure about.

The Google Doc linked from the Trello card says that the same sex outcomes for residents of the UK and Denmark should be identical to their opposite sex counterparts, with an additional "synonyms of civil partnership" section at the top. However, there are additional differences between them in this branch.

## Residents of UK

The same sex outcomes all include the "Fees" table where the opposite sex don't. Is this correct?

```
diff test/artefacts/marriage-abroad/denmark/uk/partner_british/{opposite,same}_sex.txt
diff test/artefacts/marriage-abroad/denmark/uk/partner_local/{opposite,same}_sex.txt  
diff test/artefacts/marriage-abroad/denmark/uk/partner_other/{opposite,same}_sex.txt 

# Single diff representing the changes in the three files above:

> ## Fees
> 
> Service | Fee
> -|-
> Issuing a CNI, Nulla Osta or equivalent | £65
> 
```

## Residents of Denmark

The same sex outcomes all include the address of the British embassy in Copenhagen, while the opposite sex outcomes have a link to "Make an appointment at the embassy". Is this correct?

```
diff test/artefacts/marriage-abroad/denmark/ceremony_country/partner_british/{opposite,same}_sex.txt
diff test/artefacts/marriage-abroad/denmark/ceremony_country/partner_local/{opposite,same}_sex.txt 
diff test/artefacts/marriage-abroad/denmark/ceremony_country/partner_other/{opposite,same}_sex.txt 

# Single diff representing the changes in the three files above:

< [Make an appointment at the embassy in Copenhagen.](https://www.consular-appointments.service.gov.uk/fco/#!/british-embassy-copenhagen/notice-of-marriage-or-civil-partnership/slot_picker)
---
> $A
> British Embassy Copenhagen
> Kastelsvej 36-40
> DK-2100  Copenhagen
> Denmark
> $A
> 
> $C
> Consular / passport enquiries: consular.copenhagen@fco.gov.uk
> Telephone : +45 35 44 52 00
> 
> Email: <Enquiry.Copenhagen@fco.gov.uk>
> 
> [British Embassy Copenhagen - opening hours](https://www.gov.uk/government/world/organisations/british-embassy-copenhagen/office/british-embassy)
> $C
```

File: test/artefacts/marriage-abroad/denmark/ceremony_country/partner_british/opposite_sex.txt [hunk 3]
```diff
@@ -40,6 +36,19 @@ If you or your partner have been divorced, widowed or previously in a civil part
...
+The Danish authorities will sometimes accept a ‘no trace letter’ instead of a CNI. Check with the Danish authorities to find out if they’ll accept a ‘no trace letter’.
+
+You can get a ‘no trace letter’ from the General Register Office. Ask them to make a search and quote reference ‘SEARCH 123’.
+
+$C
+**General Register Office**\\
+Telephone: +44 (0)300 123 1837 (8am to 8pm Monday to Friday and 9am to 4pm Saturday)
+$C
+
+^Your partner will need to follow the same process and pay the fees to get their own CNI.^
```
Comment (chrisroos) [inline test/artefacts/marriage-abroad/denmark/ceremony_country/partner_british/opposite_sex.txt]: It's not obvious from the Google Doc that this line should have moved in this outcome. Is it definitely correct that it's moved?

**Comment by sistephens** [comment]
LGTM 👍 

FCO are v keen for first available deploy if poss.

**Comment by pmanrubia** [comment]
Superseded by https://github.com/alphagov/smart-answers/pull/2653
